src/Screen/Fields/Matrix.php

```diff
@@ -90,6 +91,16 @@ public function maxRows(int $count)
         return $this->set('maxRows', $count);
     }
 
+    /**
+     * @param bool $value
+     *
+     * @return Field|Matrix
+     */
+    public function removableRows(bool $value)
```

Comment:
I think that constantly passing a value to a method will not be very convenient. 
Why don't we set the default value to true, just like we do in other Field methods?

Reply:
Yes, I forgot the default value in the function. I just did a commit with that.